Lesson15/Ayse/script.js

```diff
@@ -0,0 +1,504 @@
+/*
+4. Extract Domain
+   - Define a function `getDomain(email)` that uses `slice` or `substring` to
+     extract everything after '@'.
+   - Log the domain to the console.
+   - Example: "user@example.com" -> "example.com"
+*/
+console.log("Example-4");
+
+function getDomain(email) {
+  const atIndex = email.indexOf('@');
+  const domain = email.slice(atIndex + 1);
+  console.log(domain);
+  return domain;
+}
+getDomain("user@example.com");
+
+/*
+5. Check Substring
+   - Define a function `containsWord(sentence, word)` that checks if the `sentence`
+     includes `word` (use the .includes() method).
+   - If true, log: "<word> found in sentence."
+   - Else, log: "<word> not found in sentence."
+*/
+console.log("Example-5");
+
+function containWord(sentence, word) {
+
+  if (!word) {
+    console.log("Error: Missing word parameter.");
+    return;
+  }
+  if (sentence.includes(word)) {
+    console.log("<word> found in sentence.");
```

Comment:
```suggestion
    console.log(`${word} found in sentence.`);
```

Reply:
Use string interpolation to specify which word exactly (value of word param) was found in the sentence.